schemas/app-interface/app-interface-settings-1.yml

```diff
@@ -160,6 +160,20 @@ properties:
     required:
     - readTimeout
     - connectTimeout
+  glitchtip:
+    type: object
+    additionalProperties: false
+    properties:
+      readTimeout:
+        type: integer
+      maxRetries:
+        type: integer
+      mailAddress:
+        type: string
```

Comment:
Reading the code, I would call this `mailDomain`

Reply:
you're right 🤦